prereqs_linux.sh

```diff
@@ -22,11 +22,11 @@ fi
 
 # pipenv
 echo "Checking for pipenv"
-if python -c "import pipenv" &> /dev/null; then
+if python3 -c "import pipenv" &> /dev/null; then
```

Comment:
If you just have python3 installed, does this work?
I suspect the env variable will be just `python` variable, and maybe `python3 `variable will be absent. 
Not sure, needs to be investigated. @replomancer thoughts?